Tutorials/start/telemetrystart.md

```diff
@@ -1,16 +1,36 @@
-#Starting With Telemetry
+# Starting With Telemetry
 
-Now that Faraday is configured and we know Proxy is running well we can view telemetry directly in the web browser for a quick diagnostic that everything is well. Also, if you're like us at FaradayRF we think data can be beautiful :) The [Telemetry](../../Applications/Telemetry) application is located in `Applications/Telemetry` is a Flask-based server application which saves data to a SQLite database. It then exposes a RESTful API to query the database.
+Now that Faraday is configured and we know Proxy is running well we can view telemetry directly in the web browser for a quick diagnostic that everything is well. Also, if you're like us at FaradayRF we think data can be beautiful :) The [Telemetry](../../Applications/Telemetry) application is a Flask-based server which saves data to a SQLite database. It then exposes a RESTful API to query the database.
 
-Since [Telemetry](../../Applications/Telemetry) responds with JSON data we could use a web browser to print it to the screen, Python to interact with it, or an application specific to HTTP communications to interact.
+Since [Telemetry](../../Applications/Telemetry) responds with JSON data we could use a web browser to print it to the screen, Python, or any application able to query JSON responses with HTTP communications to interact.
+
+## Initializing Telemetry
+Just as we've done many times, we need to initialize the  `faraday-telemetry` configuration file. Simply run:
+* `faraday-telemetry --init-config`
+
+![Telemetry Configuration](images/faraday-telemetry-init.jpg)
 
 ## Configuring & Running Telemetry
 
-The configuration guide in [Telemetry](../../Applications/Telemetry) documentation sums up how to get the application up and running. We suggest you head over to that documentation to step through configuration. Once setup, play around with the example queries using your web browser located on the bottom of the documentation. Report back here when complete!
+To configure the `faraday-telemetry` application simply tell it what the Proxy callsign and nodeid is as programmed when we configured `faraday-proxy`:
+
+* `faraday-telemetry --callsign kb1lqc --nodeid 2`
```

Comment:
@kb1lqd are you sure you're running the latest pip version? Try `show faraday` and make sure you're using 'faraday 0.0.1014.dev1086'

I created https://github.com/FaradayRF/Faraday-Software/issues/226 to make sure this is fixed but I already fixed the original problem with this where telemetry.sample.ini had 'db.sql' instead of 'telemetrydb.sql' in the schemaname so either you're running old code or there's something else happening.